lib/util/config.js

```diff
@@ -12,10 +13,10 @@ const config = {
     /**
      * loads central properties from config file
      *
-     * @param {boolean} [silent] omit throwing errors and print messages; assuming not silent if not set
+     * @param {boolean} [skipChecks] omit throwing errors and print messages
      * @returns {object} central properties object
      */
-    async getProperties(silent) {
+    async getProperties(skipChecks = false) {
```

Comment:
again, i would say withChecks = true here to avoid double negatives. https://blog.devgenius.io/code-smell-51-double-negatives-993b6160f3e1